ibis/backends/tests/test_export.py

```diff
@@ -453,3 +449,10 @@ def test_to_torch(alltypes):
     non_numeric = alltypes.select(~selector).limit(1)
     with pytest.raises(TypeError):
         non_numeric.to_torch()
+
+
+def test_empty_memtable(backend, con):
+    expected = pd.DataFrame({"a": []})
+    table = ibis.memtable(expected)
+    result = con.execute(table)
```

Comment:
I think we only register on `execute`.

Reply:
Ok -- I remember running into trouble with this in the past, but maybe I was trying to use the expression-level `execute` instead of `con.execute`

Reply:
We do only register on execute. I suspect Gil was reading this as `table.execute()` which would use the default backend, rather than `con.execute(table)` which will always use the `con`.